test/e2e/flask/simulation-preview.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { By } from "selenium-webdriver";
+import FixtureBuilder from "../fixture-builder";
+import { unlockWallet, withFixtures, openActionMenuAndStartSendFlow } from "../helpers";
+import { Driver } from '../webdriver/driver';
+import { MOCK_RECIPIENT_ADDRESS } from "./simulation-preview.mocks";
+
+const prepareSendTransactionSimulation = async (driver: Driver, recipientAddress: string, quantity: string) => {
+  await openActionMenuAndStartSendFlow(driver);
+  await driver.fill('[data-testid="ens-input"]', recipientAddress);
```

Comment:
Can we use the `sendTransaction` helper here or is the problem that it goes too far into the flow?

Could we add an optional parameter to stop at the stage we need?

Reply:
Will address in 2nd PR.